engine/src/test/java/com/sandboni/core/engine/sta/operation/UnitTestsOperationTest.java

```diff
@@ -21,6 +21,6 @@ public void testExecuteWithExternal() {
         SetResult<TestVertex> result = operation.execute(builder.getGraph());
 
         assertNotNull(result);
-        assertEquals(0, result.get().size());
+        assertEquals(1, result.get().size());
```

Comment:
what was the reason to increase the number of tests? is because the input is now different? In that case we should create 2 tests, one with the previous and one with the new input

Reply:
The main reason is that finding external UT/Cukes operations didn't actually work, they weren't returning those tests which are marked as external.  
therefore I had to add 2 external 'dummy' tests as part of the input